src/main/java/org/openmaptiles/util/Utils.java

```diff
@@ -28,6 +44,30 @@ public static <T> T coalesce(T a, T b, T c, T d, T e, T f) {
     return a != null ? a : b != null ? b : c != null ? c : d != null ? d : e != null ? e : f;
   }
 
+  public static <T, U> T coalesceF(T a, Function<U, T> fb, U b, Function<U, T> fc, U c, Function<U, T> fd, U d,
+    Function<U, T> fe, U e, Function<U, T> ff, U f) {
```

Comment:
I wonder if it would make sense for this to just take a `Supplier<T>` so you could call it like:

```java
coalesceF(value, () -> computeValue(), () -> computeValue2(), ...)
```

Reply:
Adjusted. And looks nicer now. Thanks.